ledger/metrics/src/main/scala/com/daml/metrics/HttpJsonApiMetrics.scala

```diff
@@ -25,43 +30,76 @@ class HttpJsonApiMetrics(override val prefix: MetricName, override val registry:
   // Meters how long processing of a package upload request takes
-  val uploadPackageTimer: Timer = timer(prefix :+ "upload_package_timing")
+  val uploadPackageTimer: Timer = DropWizardMetrics.timer(prefix :+ "upload_package_timing")
   // Meters how long parsing and decoding of an incoming json payload takes
   val incomingJsonParsingAndValidationTimer: Timer =
-    timer(prefix :+ "incoming_json_parsing_and_validation_timing")
+    DropWizardMetrics.timer(prefix :+ "incoming_json_parsing_and_validation_timing")
   // Meters how long the construction of the response json payload takes
-  val responseCreationTimer: Timer = timer(prefix :+ "response_creation_timing")
+  val responseCreationTimer: Timer = DropWizardMetrics.timer(prefix :+ "response_creation_timing")
   // Meters how long a find by contract key database operation takes
-  val dbFindByContractKey: Timer = timer(prefix :+ "db_find_by_contract_key_timing")
+  val dbFindByContractKey: Timer =
+    DropWizardMetrics.timer(prefix :+ "db_find_by_contract_key_timing")
   // Meters how long a find by contract id database operation takes
-  val dbFindByContractId: Timer = timer(prefix :+ "db_find_by_contract_id_timing")
+  val dbFindByContractId: Timer = DropWizardMetrics.timer(prefix :+ "db_find_by_contract_id_timing")
   // Meters how long processing of the command submission request takes on the ledger
   val commandSubmissionLedgerTimer: Timer =
-    timer(prefix :+ "command_submission_ledger_timing")
+    DropWizardMetrics.timer(prefix :+ "command_submission_ledger_timing")
   // Meters http requests throughput
-  val httpRequestThroughput: Meter = meter(prefix :+ "http_request_throughput")
+  val httpRequestThroughput: Meter = DropWizardMetrics.meter(prefix :+ "http_request_throughput")
   // Meters how many websocket connections are currently active
-  val websocketRequestCounter: Counter = counter(prefix :+ "websocket_request_count")
+  val websocketRequestCounter: Counter =
+    DropWizardMetrics.counter(prefix :+ "websocket_request_count")
   // Meters command submissions throughput
   val commandSubmissionThroughput: Meter =
-    meter(prefix :+ "command_submission_throughput")
+    DropWizardMetrics.meter(prefix :+ "command_submission_throughput")
   // Meters package uploads throughput
-  val uploadPackagesThroughput: Meter = meter(prefix :+ "upload_packages_throughput")
+  val uploadPackagesThroughput: Meter =
+    DropWizardMetrics.meter(prefix :+ "upload_packages_throughput")
   // Meters party allocation throughput
-  val allocatePartyThroughput: Meter = meter(prefix :+ "allocation_party_throughput")
+  val allocatePartyThroughput: Meter =
+    DropWizardMetrics.meter(prefix :+ "allocation_party_throughput")
+
+  object OpenTelemetryMetrics extends OpenTelemetryFactory {
+    override val prefix: MetricName = HttpJsonApiMetrics.this.prefix
+    override val otelMeter: OtelMeter = HttpJsonApiMetrics.this.otelMeter
+  }
+
+  // golden signals
+  // http metrics
```

Comment:
Nit: We don't need this comment as the variable names already describe the grouping well.
```suggestion
```